scripts/ci/download.go

```diff
@@ -10,7 +10,7 @@ import (
 
 // download.go uses the following environment variables:
 //   URL: The url of the file to download
-//   BINDIR: The directory which the newly downloaded file will be placed
+//   DIR: The directory which the newly downloaded file will be placed
 //   FILENAME: The name the file should have after being downloaded
```

Comment:
why are we using environment variables: argument passing is much more explicit and you can always pass env variables as argument if you really really need it